openedx/features/course_experience/templates/course_experience/course-messages-fragment.html

```diff
@@ -17,14 +18,20 @@
     % for message in course_home_messages:
         <div class="course-message grid-manual">
             % if not is_rtl:
-                <img class="message-author col col-2" src="${static.url(image_src)}"/>
+                <img class="message-author col col-2" alt="Course message author" role="none" src="${static.url(image_src)}"/>
             % endif
             <div class="message-content col col-9">
                 ${HTML(message.message_html)}
             </div>
             % if is_rtl:
-                <img class="message-author col col-2" src="${static.url(image_src)}"/>
+                <img class="message-author col col-2" alt="Course message author" role="none" src="${static.url(image_src)}"/>
             % endif
         </div>
     % endfor
 % endif
+
+<%static:webpack entry="CourseGoals">
+    new CourseGoals({
+        setGoalUrl: "${set_goal_url | n, js_escaped_string}",
```

Comment:
Maybe `goalApiUrl`?

Reply:
Leaving as is because the goal API will extend to deletion, updating, etc.